internal/config/config.go

```diff
@@ -140,6 +140,22 @@ func (c *AuthConfig) Token(hostname string) (string, string) {
 	return token, source
 }
 
+// HasEnvOrConfigToken checks whether the current env or config contains a token
+func (c *AuthConfig) HasEnvOrConfigToken() bool {
```

Comment:
As we are using a dummy hostname of `example.com` is `HasEnvToken()` a more appropriate name?

Reply:
Good point, changed